src/libs/api.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+import { API_BASE_URL } from 'constants/url';
+
+const URL = axios.create({ baseURL: API_BASE_URL });
```

Comment:
URL, API_BASE_URL 뭔가 비슷해서 저는 URL이라기보다 instance라고 칭하는 편이에요!

```suggestion
const instance = axios.create({ baseURL: API_BASE_URL });
```

Reply:
의견 반영하겠습니다!